src/utils/global/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+
+import styles from './tooltip.module.scss'
+
+interface TooltipProps {
+  text: string;
+  children: React.ReactNode;
+  title: string;
+  textContainerWidth?: string;
+  leftOffset?: string;
+}
+
+const Tooltip: React.FC<TooltipProps> = (props) => {
+  const [show, setShow] = useState(false);
+
+  const tooltipStyle: React.CSSProperties = {
+    opacity: show ? 1 : 0,
+    width: props.textContainerWidth ? props.textContainerWidth : "max-content",
+    left: props.leftOffset,
+  };
+
+  return (
+    <>
+      <div
+        className={styles.wrapper}
+        aria-label={props.title}
+        onMouseEnter={() => setShow(true)}
+        onMouseLeave={() => setShow(false)}
+      >
+        <div className={styles.children}>{props.children}</div>
```

Comment:
```suggestion
        {props.children}
```
The children shouldn't need to be wrapped in another div, that seems unnecessary and just adding extra DOM levels?